exercises/concept/chessboard/.meta/config.json

```
@@ -0,0 +1,20 @@
+{
+  "authors": [
+    {
+      "github_username": "tehsphinx",
+      "exercism_username": "tehsphinx"
+    }
+  ],
+  "contributors": [
+    {
+      "github_username": "brugnara",
+      "exercism_username": "brugnara"
+    }
+  ],
```

I'm not seeing this exercise in the `main` branch currently, which would mean you get full authorship credit for porting it from elixir. Unless I'm missing something?

```suggestion
  "authors": [
    {
      "github_username": "brugnara",
      "exercism_username": "brugnara"
    }
  ],
  "contributors": [ ],
```

---

There can be more than one author of @tehsphinx  did some of the work.